src/index.js

```diff
@@ -185,10 +185,7 @@ export {
   useAbsoluteLayout,
   useFlexLayout,
 } from 'react-table';
-export {
-  default as Skeleton,
-  SkeletonTheme,
-} from 'react-loading-skeleton';
+export { default as Skeleton, SkeletonTheme } from './Skeleton/index';
```

Comment:
nit: this change wasn't really necessary. also, the `index` is inferred like the other imports of `./Bubble` and `./Dropzone`.